parse_iwyu_logs.py

```diff
@@ -15,33 +15,37 @@
 }
 </style>
 </head>"""
-print '<a href='+'"'+ sys.argv[1].split('/')[-1] + '"' + '>' + Access BuildLog + '</a><br/>'
+print '<a href='+'"'+ sys.argv[1].split('/')[-1] + '"' + '>' + 'Access BuildLog' + '</a><br/>'
 print '<table  align="center">'
 for l in fd:
-  if 'remove' in l:
+  if 'remove these lines' in l:
     sec=iter(fd)
     line=sec.next()
-    if line.startswith('-'):
-      print '<tr><td bgcolor="#00FFFF"><h2>'+l[70:].replace('lines','includes')+'</h2>'
-      while line.startswith('-'):
-        line=line.replace('<','')
-        line=line.replace('>','')
-        line=line.replace('"','')
-        line=line.replace('- #include ','')
-        print '<br/>'+line.split('//')[0]
+    line=line.rstrip()
+    if len(line):
+      print '<tr><td bgcolor="#00FFFF"><h2>'+'/'.join(l.split('/')[-4:])+'</h2>'
```

Comment:
wrong assumption here. File could be in sub-directory e.g src/Subsystem/Package/src/subdir/file.cc
[-4:] will return /Package/src/subdir/file.cc which is wrong